src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -102,7 +104,7 @@
     /**
      * The map of Pub-Sub nodes to videobridge JIDs.
      */
-    private final Map<String, String> pubSubToBridge = new HashMap<>();
+    private final Map<String, String> pubSubToBridge = new ConcurrentHashMap<>();
```

Comment:
2 out of 3 uses are protected by a synchronized on this, 1 out of 3 is not protected & it's in the init method of the instance (which presumably makes the instance usable). I'm not sure what practical advantages ConcurrentHashMap brings.